### DISCUSSION TITLE
[REQUEST] Gitlab  LXC

### DISCUSSION THREAD

**Discussion by paloky** [tteck/Proxmox#1169] [Ideas]
Hi.

It would be great if a GitLab container could be made.

Thank's a lot.

**Comment by azat385** [comment]
if it is possible or suggest anything else please!

**Comment by tteck** [comment]
Use the TurnKey LXC Appliances script and select GitLab

**Reply by beautyfree** [reply]
can you explain how to install it?
i have installed turnkey with gitlab but when i open ip of container in browser it cant get access (btw i can open webadmin of turnkey on 12321 port)

**Reply by tteck** [reply]
You need to go to the console to finish the setup

![Screenshot 2024-03-15 4 48 49 PM](https://github.com/tteck/Proxmox/assets/21342670/109cbd1d-30ed-49ce-9d16-bfff76822724)
